tests/api/v1/test_teams.py

```diff
@@ -860,7 +860,7 @@ def test_api_user_without_team_challenge_interaction():
         gen_flag(app.db, 1)
 
         with login_as_user(app) as client:
-            assert client.get("/api/v1/challenges").status_code == 403
+            assert client.get("/api/v1/challenges").status_code == 200
```

Comment:
Why did this change? 

I don't necessarily disagree but I don't think this PR would be the right place to change this.